Shared/include/Shared/Buffer.hpp

```diff
@@ -14,9 +14,7 @@ class Buffer : public Vector<uint8>, Unique
 	Buffer(Buffer&& rhs);
 	Buffer& operator=(Buffer&& rhs);
 	// Creates a new buffer from a string, without the terminating null character
-	Buffer(const char* string);
-	// Creates a new buffer with an initial size
-	Buffer(size_t initialSize);
```

Comment:
msvc complains that this is indistinguishable from the constructor from `std::vector`. Since they do the same job (sans initialization), I just removed this constructor.